charts/stable/qbittorrent/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: v4.3.5
+appVersion: v4.3.7
 description: qBittorrent is a cross-platform free and open-source BitTorrent client
 name: qbittorrent
-version: 12.0.0
+version: 13.0.0
```

Comment:
It looks like the changes are not breaking, so according to semver 12.1.0 would probably be better.

Reply:
ok